pywbem/_subscription_manager.py

```diff
@@ -794,6 +810,10 @@ def add_filter(self, server_id, source_namespace, query,
                 raise ValueError(
                     _format("Filter ID contains ':': {0!A}", filter_id))
 
+        if name is not None and owned:
+            raise ValueError("The name parameter cannot be used to add "
+                             "owned filters.")
+
```

Comment:
I think the combination of using filter_id and not owned should also be raised as an error.

Reply:
I will change this one.